### DISCUSSION TITLE
Using MSCognitiveSpeechForVoiceAttack with EDDI

### DISCUSSION THREAD

**Discussion by Solamain** [EDCD/EDDI#2414] [EDDI VoiceAttack Q&A] [answered]
Was just curious if there is any chance you might integrate https://github.com/jamescl604/MSCognitiveSpeechForVoiceAttack with EDDI, it would be great to have all of those voices to use, and that project brings them to VoiceAttack, but I do not see a way to use them in EDDI.

**Comment by Tkael** [comment, marked as answer]
I am interesting in doing something similar for EDDI eventually but have been focusing in other areas recently.
EDDI may also not benefit very much from caching, given the highly dynamic nature of the responses generated by the Speech Responder.
